packages/@webex/media-helpers/test/unit/spec/sendslots-manager.ts

```diff
@@ -0,0 +1,195 @@
+import SendSlotsManager from "../../../src/sendslots-manager";
+import { MediaType, MultistreamRoapMediaConnection } from "@webex/internal-media-core";
+import {expect} from '@webex/test-helper-chai';
+import sinon from 'sinon';
+
+describe.only('SendSlotsManager', () => {
+    let sendSlotsManager: SendSlotsManager;
+    const LoggerProxy = {
+        logger: {
+            info: sinon.stub(),
+        },
+    };
+    
+    beforeEach(() => {
+        sendSlotsManager = new SendSlotsManager(LoggerProxy);
+    });
+    
+    describe('createSlot', () => {
+        let mediaConnection;
+        const mediaType = 'AUDIO_MAIN' as MediaType;
```

Comment:
I think you can just use `MediaType.AudioMain` here instead of having to cast it. Same with a few other places in this file.